mvebu64boot.c

```diff
@@ -751,7 +753,11 @@ static int open_tty(const char *device)
 	int flags;
 	int fd;
 
-	printf("Opening tty device '%s'\n", device);
+	if (strcmp(device, "-") == 0) {
+		return atoi(device+1);
```

Comment:
`return`, really? It is purely broken as code below which setups console for bootrom is skipped.